playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -1,3 +1,13 @@
+- name: Configure ntp
+  hosts:
+    - oo_nodes_to_config
+    - oo_etcd_to_config
+  vars:
+    ntp_require_clock_sync: "{{ openshift_require_clock_sync | default(false) | bool }}"
+  roles:
+    - role: ntp
```

Comment:
We'll want to sync the clocks prior to the additional_config.yml playbook being run.

One of the patterns that we've started to use lately (to clean up the steps required in playbooks as well as make it easier to consume the roles outside of playbooks) is to configure things like this as a role dependency.

The way we are now pulling in the docker role is one example (and @abutcher is working on implementing it as part of his delegate_to refactoring: https://github.com/openshift/openshift-ansible/pull/1501)

An example of the way it works is as follows:
- openshift_node defines a dependency on the openshift_docker role: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/meta/main.yml
- the openshift_docker role only consists of a metadata file which describes  dependency on openshift_docker_facts and docker
- the openshift_docker_facts role persists local facts related to openshift docker configuration and uses set_fact to map the openshift provided variables/fact defaults to variables that the docker role understands.

The main reason for having openshift_docker and openshift_docker_facts is to make sure that we are setting the variables before the docker role is run, since the role dependencies are executed before the dependent role. That type of wrapping wouldn't be needed unless consuming an upstream ntp/chrony role that was meant to be generic.

For ntp config, I'd say we want to add it as a dependency to at least the following roles: openshift_etcd (coming with Andrew's delegate_to changes), openshift_master and openshift_node to start.

One approach that may work is a role called openshift_clock_sync or even just openshift_clock (to align naming with openshift_env set variables) that would conditionally set dependencies on an ntp or the chrony role accordingly (the when clause works in the dependency field of the meta/main.yml file). It might even be worth it to use an upstream ntp role like https://galaxy.ansible.com/geerlingguy/ntp/ or https://gitlab.com/MathieuMD/ansible-role-ntp